pylint/checkers/stdlib.py

```diff
@@ -52,6 +52,11 @@
 OPEN_MODULE = "_io"
 
 DEPRECATED_ARGUMENTS = {
+    (3, 7, 0): {
+        "int": ((None, "x"),),
```

Comment:
I don't understand that one. There was a ``x`` argument to int, and it was removed ? Could you provide some documentation ?

Reply:
Of course:
> Functions bool(), float(), list() and tuple() no longer take keyword arguments. The first argument of int() can now be passed only as positional argument. [1] 

Slightly problem is that it is not deprecation but removal. Maybe this can be raised as warning for all versions of python.

BTW. the `list` and `tuple` is a mistery for me because I was not able to replicate it in python 3.6:

* python 3.6:
```python
Python 3.6.13 (default, Feb 20 2021, 21:40:46)
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> int(x=1)
1
>>> float(x=3)
3.0
>>> list(iterable=[])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'iterable' is an invalid keyword argument for this function
>>> tuple(iterable=[])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'iterable' is an invalid keyword argument for this function
>>>
``` 
* python 3.7:
```
Python 3.7.10 (default, Feb 20 2021, 21:15:28)
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> int(x=1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'x' is an invalid keyword argument for int()
>>> float(x=3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: float() takes no keyword arguments
>>> tuple(iterable=[])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: tuple() takes no keyword arguments
>>> list(iterable=[])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: list() takes no keyword arguments
```

[1] https://docs.python.org/3/whatsnew/3.7.html#api-and-feature-removals

Reply:
Thank you, I failed to find that doc myself.